mgmt/rest/metric.go

```diff
@@ -171,7 +174,11 @@ func (s *Server) getMetricsFromTree(w http.ResponseWriter, r *http.Request, para
 	respond(200, b, w)
 }
 
-func respondWithMetrics(host string, mets []core.CatalogedMetric, w http.ResponseWriter) {
+func respondWithMetrics(host string, mets []core.CatalogedMetric, w http.ResponseWriter, version string) {
+	if version == "v2" {
+		apiV2.RespondWithMetrics(host, mets, w)
+		return
+	}
```

Comment:
I would disagree. I think now (when we are adding the first additional version) is the appropriate time to think about how future versions will be added and how it can be done cleanly.